templates/blog/post_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block content %}
+  
+  <h1>{{ post.title }}</h1>
+  <p>Автор: {{ post.owner.username }}</p>
+  <p>Створено: {{ post.created_time|date:"d.m.Y H:i" }}</p>
+  <p>{{ post.content }}</p>
+
+  <h2>Коментарі ({{ post.comment_count }})<a style="float: right" href="{% url 'blog:create' post.id %}">+</a></h2>
```

Comment:
The URL in the href attribute for the 'create' link seems to be incorrect. It should likely point to a comment creation view, but currently, it uses 'blog:create', which might not exist or be incorrect. Please verify the URL name.